src/drawer.css.ts

```diff
@@ -0,0 +1,112 @@
+import {style} from '@vanilla-extract/css';
+import * as mq from './media-queries.css';
+import {vars} from './skins/skin-contract.css';
+import {sprinkles} from './sprinkles.css';
+
+export const ANIMATION_DURATION_MS = 400; // review
+
+export const container = style([
+    sprinkles({
+        position: 'fixed',
+        display: 'flex',
+        background: vars.colors.background,
+        top: 0,
+        right: 0,
+        bottom: 0,
+        overflow: 'hidden',
+    }),
+    {
+        paddingBottom: 'env(safe-area-inset-bottom)',
+        '@media': {
+            [mq.mobile]: {
+                left: 0,
+                transition: `transform ${ANIMATION_DURATION_MS}ms cubic-bezier(0.32, 0.72, 0, 1)`,
+            },
+            [mq.tabletOrBigger]: {
+                borderTopLeftRadius: vars.borderRadii.container,
+                borderBottomLeftRadius: vars.borderRadii.container,
+                transition: `transform ${ANIMATION_DURATION_MS}ms cubic-bezier(0.65, 0, 0.35, 1)`,
```

Comment:
```suggestion
                transition: `transform ${ANIMATION_DURATION_MS}ms cubic-bezier(.5,0,.5,1)`,
```

Tested in local, I feel this curve is more natural and smooth

Reply:
ok. Please update specs